### DISCUSSION TITLE
"prisma migrate dev" error due to statement timeout?

### DISCUSSION THREAD

**Discussion by Toryn02** [supabase/supabase#11250] [Questions]
Hi all,

I just started a new project attempting to use Supabase with Prisma.  When I attempt to sync my initial Prisma schema with my Supabase database using `npx prisma migrate dev`, however, I am getting the following error:

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "db.xxxxxxxxxxx.supabase.co:5432"

Error: db error: ERROR: canceling statement due to statement timeout
   0: migration_core::state::DevDiagnostic
             at migration-engine/core/src/state.rs:264
```

I'm brand new to using Supabase with Prisma so I very well may be doing something incorrect here.  I very much appreciate any advice that someone can give me.

Thanks very much,
Josh

**Comment by Phillipjacob** [comment]
I'm getting the same error when removing the shadowDatabaseUrl in the .env file.

Might be helpful: https://github.com/prisma/prisma/issues/16853#issuecomment-1374830020

**Reply by Phillipjacob** [reply]
Also, make sure your user is superuser, Supabase changed that on all projects. 
Go to your sql editor and write: alter user postgres with superuser;
